integrated_channels/canvas/client.py

```diff
@@ -134,16 +138,21 @@ def create_course_completion(self, user_id, payload):  # pylint: disable=unused-
                 break
 
         if not course_id:
-            raise CanvasClientError(
+            return (
+                HTTPStatus.NOT_FOUND.value,
```

Comment:
Yeah, similar situation as with my try/catch on the client errors above this- was trying to just surface the response code and message rather than raise the error but now we catch those so this should just be a raise. I'll change it to Client Error.